tests/integration/create_token.py

```diff
@@ -63,6 +65,6 @@ def create_token_with_survey_url(self, form_type_id, eq_id, survey_url, **extra_
         return self.generate_token(payload_vars)
 
     def generate_token(self, payload):
-        token = encode_jwt(payload, self._upstream_kid, self._secret_store, KEY_PURPOSE_AUTHENTICATION)
+        token = JWTHelper.encode(payload, self._upstream_kid, self._secret_store, KEY_PURPOSE_AUTHENTICATION)
 
-        return encrypt_jwe(token, self._sr_public_kid, self._secret_store, KEY_PURPOSE_AUTHENTICATION)
+        return JWEHelper.encrypt(token, self._sr_public_kid, self._secret_store, KEY_PURPOSE_AUTHENTICATION)
```

Comment:
Likewise, `encode` -> `encrypt` can be dropped in favour of `sdc.crypto.encrypter.encrypt`.

Reply:
Attached suggested patch [here](https://github.com/ONSdigital/eq-survey-runner/files/1333883/helper.txt).